src/redux/movieSelect.test.ts

```diff
@@ -0,0 +1,17 @@
+import { mockMovie } from 'mocks/movies';
+import moviesSlice, { MoviesState } from './movies';
+
+const state: MoviesState = {
+  movies: [mockMovie],
+  moviesCount: 1,
+};
+
+describe('movieSelect', () => {
+  test('returns correct value', () => {
+    const movieState = moviesSlice.reducer(
+      state,
```

Comment:
Whenever you declare new test, make sure that variables you are using outside of test declaration are **always recreated**. Why? When you have test that might modify your data, your variable value might be also changed and it could lead to some unpredicted tests behaviour. In this case, make sure that `state` is always recreated (use spread-operator for this)